### ISSUE TITLE
Invariant Violation: requireNativeComponent: "DatePickerManager" was not found in the UIManager. TypeError: null is not an object (evaluating '_reactNative.NativeModules.RNDatePicker.openPicker')

### ISSUE THREAD

**Issue by yesIamFaded** [henninghall/react-native-date-picker#454]
So there are 2 Errors using this Module. I am using Expo.

First using Inline : 

```
            <DatePickerNew                                                                              
              date={this.state.activityDate}
              onDateChange={(date) => {
                console.log(date)
                this.setState({
                  activityDate: date
                })
              }}
            /> 
```

- the code does not work, as soon as my app loads it crashes with this error: 
- Invariant Violation: requireNativeComponent: "DatePickerManager" was not found in the UIManager
- I have installed the Module as it is stated in the docs

When I use the modal way :

```
            <DatePickerNew                                                                
              modal
              open={this.state.showDatePicker}
              date={this.state.activityDate}
              onDateChange={(date) => {
                this.resetSOFT()             
                this.setState({ 
                  // activityDate: date,
                  activityDate: Moment(date).format("YYYY-MM-DD"),
                  checked: false,
                })
              }}
              onCancel={() => {
                this.setState({
                  showDatePicker: false
                })
              }}
              onConfirm={() =>{
                this.setState({
                  showDatePicker: false
                })
              }}
            />

```

- it also does not work, as soon as I press the button to show the modal it crashes with this error:
-  TypeError: null is not an object (evaluating '_reactNative.NativeModules.RNDatePicker.openPicker')

**Smartphone (please complete the following information):**
 - OS: Android Emulator 
 - React Native version Expo SDK v42.0.0
 - react-native-date-picker version tried 4.1.3 and currently 4.1.2

Basically I need some help - I have tried 4 different pickers and all of them have major problems except the oldest one wich is not maintained anymore hence my swap.

**Comment by henninghall** [comment]
I'have tried but I cannot reproduce it. 

Can you reproduce it when doing the following?

1. `expo init my-project`
2. `cd my-project`
3. `expo install react-native-date-picker expo-dev-client`
4. Add the code to display the DatePicker in `App.js`
5. `expo run:android`

**Comment by henninghall** [comment]
Duplicate of https://github.com/henninghall/react-native-date-picker/issues/404

**Comment by ghyath5** [comment]
Re build your app `yarn android` this will fix the error
